backend/corpora/common/entities/entity.py

```diff
@@ -88,19 +88,17 @@ def _create_sub_objects(
 
         Another use would be to overwrite column specified in the rows.
 
-       :return: a list of database objects to create or modify.
+       :return: a list of database objects to create.
         """
         add_columns = add_columns if add_columns else {}
         db_objs = []
         for columns in rows:
-            _columns = dict(**columns, **add_columns)  # if there are matching keys in columns and add_columns,
-            # the key value in add_columns will be used.
-            _id = _columns.get("id")
-            if _id:
-                primary_key = (_id, *primary_keys) if primary_keys else _id
-                row = cls.db.get(db_table, primary_key)
-            else:
-                _columns["id"] = str(uuid.uuid4())
-                row = db_table(**_columns)
+            #  if there are matching keys in columns and add_columns,
+            #  the key value in add_columns will be used.
```

Comment:
Nit: can be combined with previous line?

Reply:
fixed